test/models/job_execution_test.rb

```diff
@@ -139,6 +139,7 @@
     lines.must_include "DEPLOYER_NAME=John Doe"
     lines.must_include "REFERENCE=master"
     lines.must_include "REVISION=#{job.commit}"
+    lines.must_include "COMMIT_RANGE=#{deploy.changeset.commit_range}"
```

Comment:
better spell it out so we can see what it will actually look like

Reply:
Not sure I follow... the actual commits depend on the test repo so they cannot be hardcoded, right?

Since there is only one deploy in the test I've changed it to expect a commit range beginning and ending with `job.commit`, how does that sound?